src/Installers/Windows/GenerateNugetPack.ps1

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+param(
```

Comment:
Not a blocker but I'd rather have such a generic script in eng/scripts/

Reply:
It isn't very generic, they are both used for specifically generating a nuget file with mis/cabs. I'll rename the script though to be more specific.